source/Pages/best_practices.rst

```diff
@@ -0,0 +1,160 @@
+.. _best-practices:
+*****************
+Best practices
+*****************
+
+.. Tip:: This is a best practices section to help you achieve maximum performance for your jobs on Spider. In this page you will learn which options are most efficient for:
+
+     * your software installation on Spider
+     * storing your data on the platform
+     * managing your data on Spider  
+     * running a large amount of jobs on Spider
+
+
+==========     
+Background
+==========
+
+Spider is continuously expanding as a unique data processing and collaboration platform. The growing demand for storage space, the diversity in applications and the various ways the system is used, all bring some technical challenges. For example, if you're doing a lot of IO-operations (reading and writing files) in your workflows, you should be mindful where these operations are performed as some options can significantly affect the performance of your jobs and the system load. 
+
+Our local file system on Spider is CephFS and is suitable as a staging area for your data before or after analysing it. CephFS hosts both your home and project directories on Spider. It is designed for efficient IO of large files, but when dealing with many small files the file system performance can be degraded. This is because CephFS relies on a parallel distributed system that involves many disks to store the data itself and metadata servers to store the files metadata. As a result, when you operate on many small files or run code from python environments, the system response can become slow for you and other users on the platform. 
+
+In order to improve the performance of scientific workflows on Spider's storage, we have prepared this best practices guide that includes several tips to install, store and analyse your data efficiently.
+
+
+.. _software-practices:
+
+Software installation practices
+===============================
+
+If your software loads a large number of small files upon execution in your jobs then you may see poor I/O performance even if the total software size is not that big. There are ways to mitigate potential bottlenecks, such as the use of CVMFS or containers technology.
+
+Here is an overview of the features and capabilities of some of the software installation options supported on Spider followed by some general guidelines:
+
+==============================================   ==============   ==============   =============   ======================
+Feature                                          CephFS           Apptainer        Lumi            Softdrive
```

Comment:
Do you mean LUMI container wrapper? I think here should refer to the tool, instead of LUMI.

Reply:
Good point. I'll fix it